index.html

```diff
@@ -13278,6 +13280,36 @@ <h2>Relationships in the Accessibility Tree</h2>
   &lt;div role="listitem"&gt;The "list" is my accessibility parent.&lt;/div&gt;
 &lt;/div&gt;
           </pre>
+
+		  <section id="tree_groups">
+			<h3>Calculating Group Position in the Accessibility Tree</h3>
+			<p>When <pref>aria-level</pref>, <pref>aria-posinset</pref>, or <pref>aria-setsize</pref> are defined on supported roles, they allow authors to override the default group position and level calculation. When those attributes are not specified by the author, user agents MUST compute them following the rules in this section.</p>
+
+			<p>If <pref>aria-level</pref> is not specified on <rref>comment</rref>, <rref>row</rref> within a <rref>treegrid</rref>, or <rref>treeitem</rref>, user agents MUST compute the level as follows:</p>
```

Comment:
I don't like listing roles but I don't see a good alternative. The best I can come up with "when not required by the role".